Source/Atomic/UI/UIEditField.cpp

```diff
@@ -213,6 +215,17 @@ void UIEditField::OnFocusChanged(bool focused)
             if (!w->GetMultiline())
                 styleEdit->selection.SelectAll();
             firstFocusFlag_ = true;
+
+#if defined(ANDROID) || defined( __ANDROID__) || defined(IOS)
+
```

Comment:
For reference, we  have platform defines which help keep things consistent: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/CMakeLists.txt#L106

```ATOMIC_PLATFORM_ANDROID``` and ```ATOMIC_PLATFORM_IOS```, there is also ```ATOMIC_PLATFORM_LINUX```, ```ATOMIC_PLATFORM_WINDOWS```, etc